pkg/router/template/router.go

```diff
@@ -790,3 +823,42 @@ func generateCACertKey(config *ServiceAliasConfig) string {
 func generateDestCertKey(config *ServiceAliasConfig) string {
 	return config.Host + destCertPostfix
 }
+
+// routeKeys returns the internal router keys to use for the given Route.
+// A route can have several services that it can point to, now
+func routeKeys(route *routeapi.Route) ([]string, []int32) {
```

Comment:
There's already a `routeKey` method in this file - could be confusing `*Key` vs `*Keys` but its minor.